pkg/options/options.go

```diff
@@ -110,6 +110,11 @@ var ImagePullSecret = ""
 // pod)
 var MiniEnv = false
 
+// MiniEnv setting this option indicates to the operator that it is deployed on low resource environment
```

Comment:
```suggestion
// DevEnv setting this option indicates to the operator that it is deployed on low resource environment
```

Reply:
Please update the text (and not only `MiniEnv` to `DevEnv`).